### PR TITLE
Move Desktop Enterprise docs

### PR DESCRIPTION
**Pull request by usha-mandya**
- Moved Desktop Enterprise topics to a new location `/desktop/enterprise` as there are plans to set a global redirects to move Enterprise docs to the Mirantis docs site.
- Added redirects for Desktop Enterprise docs

### CODE REVIEW

**Comment by usha-mandya** [comment]
@thaJeztah PTAL - https://deploy-preview-10658--docsdocker.netlify.app/desktop/enterprise/

CC @nebuk89

File: desktop/enterprise/admin/install/mac.md [hunk 1]
```diff
@@ -32,7 +34,7 @@ Double-click the `.pkg` file to begin the installation and follow the on-screen
 
 Mac administrators can use the command line option `\$ sudo installer -pkg Docker.pkg -target /` for fine tuning and mass installation. After running this command, you can start Docker Desktop from the Applications folder on each machine.
 
-Administrators can configure additional settings by modifying the administrator configuration file. For more information, see [Configure Desktop Enterprise for Mac](/ee/desktop/admin/configure/mac-admin).
+Administrators can configure additional settings by modifying the administrator configuration file. For more information, see [Configure Desktop Enterprise for Mac](/desktop/enterprise/admin/configure/mac-admin).
```
Comment (thaJeztah) [inline desktop/enterprise/admin/install/mac.md:37]: nit: these URLs should have a trailing `/` - saw a couple of other ones as well (we can update in a follow-up and link to the `.md` file)

File: _data/toc.yaml [hunk 2]
```diff
@@ -1503,37 +1503,37 @@ manuals:
       nosync: true
 - sectiontitle: Docker Desktop Enterprise
```
Comment (thaJeztah) [inline _data/toc.yaml:1504]: Should we move this under `Desktop` in the menu?

File: desktop/enterprise/admin/install/mac.md (hunk 1 above)
Comment (usha-mandya) [reply desktop/enterprise/admin/install/mac.md:37]: Sure, I only changed the initial path. Happy to follow up in a separate commit.

**Comment by thaJeztah** [comment]
Let's bring this one in 👍

**Pull request merged**